apps/wallet-mobile/src/features/Receive/common/ShareQRCodeCard/ShareQRCodeCard.tsx

```diff
@@ -0,0 +1,176 @@
+import React, { RefObject, useEffect, useRef, useState } from 'react'
+import { StyleSheet, TouchableOpacity, View } from 'react-native'
+import QRCode from 'react-native-qrcode-svg'
+import Animated, { FadeInDown, FadeOutDown, Layout } from 'react-native-reanimated';
+import Share from 'react-native-share';
+import ViewShot, { captureRef } from "react-native-view-shot";
```

Comment:
These dependencies are missing in package.json

Reply:
please run `yarn workspace @yoroi/wallet-mobile add react-native-share react-native-view-shot` in the project root